docs/vaccel_go.md

```diff
@@ -26,101 +26,56 @@ about Go installation can be found [here](https://go.dev/doc/install),
 but there are also instructions on how to install `Go` in the
 [vAccel-go bindings installation guide](https://github.com/nubificus/go-vaccel).
 
-### vAccel Go package
+## vAccel Go package
 The vaccel package in Golang provides access to vAccel operations, which
 can be used by the developers on their own `Go` programs. The vaccel package uses 
-the native `C` bindings in order to use the vAccel `C` API. The following diagram 
-demonstrates the functionality of the vaccel package:
+the native `C` bindings in order to use the vAccel `C` API.
 
-<figure>
-  <img src="img/vaccel_go.png" align=center />
-  <figcaption>Figure 1: High-level overview of the vAccel Go package</figcaption>
-</figure>
+## Installation Guide
 
-### Installation Guide
-#### vAccel Installation
+### vAccel Installation
 First of all, a vAccel installation is required before proceeding to the next sections. 
-
-#### Build from source
-In Ubuntu-based systems, you need to have the following packages to build vaccelrt:
-1. `cmake`
-2. build-essential
-
-You can install them using the following command:
-```
-sudo apt-get install -y cmake build-essential
-```
-Get the source code for **vaccelrt**:
+You can install `vaccel` using the following commands:
 ```
+sudo apt update
+sudo apt install --no-install-recommends python3-pip build-essential cargo pkg-config ninja-build libclang-dev
+sudo pip install meson
+
 git clone https://github.com/nubificus/vaccel --recursive
+cd vaccel
+meson setup -Dauto_features=enabled build
+meson compile -C build
+meson install -C build
```

Comment:
Instead of duplicating instructions here maybe link to build instructions and binary installation pages